### ISSUE TITLE
add no wrap in buffer 

### ISSUE THREAD

**Issue by nda-cunh** [Donaldttt/fuzzyy#47]
![image](https://github.com/user-attachments/assets/5ed9e96b-6c60-4a60-abe6-84b5b7bb1f2f)

with FuzzyFiles , print files content with `set nowrap`

**Comment by Donaldttt** [comment]
Then some text might be invisible. Are you suggesting to also add hotkey to navigate around the preview window?

**Comment by nda-cunh** [comment]
it's a preview window so it doesn't really matter, does it?

**Comment by mmrwoods** [comment]
This might not matter so much for finding files (e.g. FuzzyFiles), but it does matter for finding in files (e.g. FuzzyGrep), so, IMHO anyway, it would not make sense to disable wrap for all preview windows.

It might make sense as a window layout option though, allowing wrap to be set to `false` for the preview window if desired.

**Comment by nda-cunh** [comment]
I've disabled it in the file search only, but I'm not sure if adding this option is really useful anymore :)

**Comment by mmrwoods** [comment]
> I've disabled it in the file search only, but I'm not sure if adding this option is really useful anymore :)

How have you disabled it in the file search only? By modifying the code or adding your own `:FuzzyFiles` command (this should be quite easy if that's not what you've done, add a function that calls `files.start()`, capture the return value, a dict of window ids, use that to modify the properties of the preview popup).

**Comment by nda-cunh** [comment]
I have modify the code, but maybe I'll go code my fuzzyfile for sort and remove .o and other folder directly in Selector

**Comment by nda-cunh** [comment]
yes but it work only with ripgrep but in my school we have only grep by default

**Comment by mmrwoods** [comment]
Are you sure `git-grep` isn't being invoked here rather than `grep`? `g:fuzzyy_exclude_file` and `g:fuzzyy_exclude_dir` should work fine with `grep`, but if you are in a git repo, `git-grep` is normally used instead for performance reasons, and `git-grep` does not support custom excludes supplied as command line arguments (but it will obviously respect `.gitignore`).

You can explicitly disable the fallback to `git-grep` by setting `g:fuzzyy_respect_gitignore` to false/0.
